### PR TITLE
Release IIB v6.8.0

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

**Comment by yashvardhannanavati** [comment]
@release-engineering/exd-guild-hello-operator Could you PTAL?

**Review by JAVGan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.8.0
+- Add static types to IIB
```
Comment (chandwanitulsi) [inline CHANGELOG.md:10]: Few more PRs can be added in the changelog, like https://github.com/release-engineering/iib/pull/455 and https://github.com/release-engineering/iib/pull/451
Also, static types change was already released no?

Comment (chandwanitulsi) [reply CHANGELOG.md:10]: I just merged https://github.com/release-engineering/iib/pull/461 as well

Comment (lipoja) [reply CHANGELOG.md:10]: - Remove misleading log from API response
- Unsupported retry package replaced by tenacity

Comment (yashvardhannanavati) [reply CHANGELOG.md:10]: Made the changes

**Review by lipoja — APPROVED** [review]

**Pull request merged**